### PR TITLE
[PIL-996][PS]Hide UK in CountryList and UK postcode validation

### PR DESCRIPTION
**Pull request by prsathmrc**

### CODE REVIEW

File: src/test/resources/features/RFMGRSflowPages.feature [hunk 1]
```diff
@@ -71,8 +71,11 @@ Feature: RFM ultimate parent entity and New nominated Filling Member GRS journey
       | KEY          | VALUE          |
       | addressLine1 | Address Line 1 CYA |
       | addressLine3 | City CYA          |
-      | postalCode   | EH5 5WY        |
-      | countryCode  | Australia |
+      | postalCode   | INVALID       |
+      | countryCode  | United Kingdom |
+    Then I should see address error message Enter a valid UK postcode or change the country you selected on the Postal code Element
```
Comment (srikanthhmrc) [inline src/test/resources/features/RFMGRSflowPages.feature:76]: please move error validation tests to unit tests and remove from ATs

File: src/test/resources/features/RFMnfmNoIDFlow.feature [hunk 2]
```diff
@@ -45,12 +45,11 @@ Feature: RFM CYA - NFM No ID flow
     When I refresh the page
     When I provide RFM New NFM Name as Test CYA
     Then I should navigate to RFM New NFM Contact Address Page
-    And I enter Country as United Kingdom
+    And I enter Country as United Arab Emirates
     And I click on Country selected
     And I click on Continue button
     Then I should see address error message Enter the first line of the address on the Address Line Element
```
Comment (srikanthhmrc) [inline src/test/resources/features/RFMnfmNoIDFlow.feature:51]: same here:
please move error validation tests to unit tests and remove from ATs

File: src/test/resources/features/CreateRegAndSubE2ETests.feature [hunk 3]
```diff
@@ -507,8 +508,11 @@ Feature: User registration and subscription e2e journey
       | KEY          | VALUE          |
       | addressLine1 | Address Line 1 |
       | addressLine3 | City           |
-      | postalCode   | EH5 5WY        |
+      | postalCode   | INVALID       |
       | countryCode  | United Kingdom |
+    Then I should see address error message Enter a valid UK postal code or change the country you selected on the Postal code Element
```
Comment (srikanthhmrc) [inline src/test/resources/features/CreateRegAndSubE2ETests.feature:513]: please move error validation tests to unit tests and remove from ATs

File: src/test/resources/features/CreateRegAndSubE2ETests.feature [hunk 4]
```diff
@@ -584,8 +589,11 @@ Feature: User registration and subscription e2e journey
       | KEY          | VALUE          |
       | addressLine1 | Address Line 1 |
       | addressLine3 | City           |
-      | postalCode   | EH5 5WY        |
+      | postalCode   | INVALID       |
       | countryCode  | United Kingdom |
+    Then I should see address error message Enter a full UK postcode on the Postal code Element
```
Comment (srikanthhmrc) [inline src/test/resources/features/CreateRegAndSubE2ETests.feature:594]: same here:
please move error validation tests to unit tests and remove from ATs

**Review by srikanthhmrc — CHANGES_REQUESTED** [review]

File: src/test/resources/features/ContactDetailsAndRegistration.feature [hunk 5]
```diff
@@ -15,7 +15,7 @@ Feature: Contact details for the filing member
       | addressLine1 | Address Line 1 |
       | addressLine3 | City           |
       | postalCode   | EH5 5WY        |
-      | countryCode  | United Kingdom |
+      | countryCode  | United Arab Emirates |
```
Comment (srikanthhmrc) [inline src/test/resources/features/ContactDetailsAndRegistration.feature:18]: adjust the format spaces

File: src/test/resources/features/ContactDetailsAndRegistration.feature [hunk 6]
```diff
@@ -164,7 +164,7 @@ Feature: Contact details for the filing member
       | Address                         | Address Line 1           |
       | Address                         | City                     |
       | Address                         | EH5 5WY                  |
-      | Address                         | United Kingdom           |
+      | Address                         | United Kingdom     |
```
Comment (srikanthhmrc) [inline src/test/resources/features/ContactDetailsAndRegistration.feature:167]: adjust the format spaces

File: src/test/resources/features/ContactDetailsAndRegistration.feature [hunk 7]
```diff
@@ -259,7 +258,7 @@ Feature: Contact details for the filing member
       | addressLine1 | Address Line 1 |
       | addressLine3 | City           |
       | postalCode   | EH5 5WY        |
-      | countryCode  | United Kingdom |
+      | countryCode  | United Arab Emirates  |
```
Comment (srikanthhmrc) [inline src/test/resources/features/ContactDetailsAndRegistration.feature:261]: adjust the format spaces

File: src/test/resources/features/ContactDetailsAndRegistration.feature [hunk 8]
```diff
@@ -274,7 +273,7 @@ Feature: Contact details for the filing member
...
```
Comment (srikanthhmrc) [inline src/test/resources/features/ContactDetailsAndRegistration.feature:276]: adjust the format spaces

**Review by srikanthhmrc — CHANGES_REQUESTED** [review]

File: src/test/resources/features/RFMnfmNoIDFlow.feature (hunk 2 above)
Comment (prsathmrc) [reply src/test/resources/features/RFMnfmNoIDFlow.feature:51]: This is the flow now So there wouldn't be United kingdom anymore so need to change it to different country.

**Review by srikanthhmrc — APPROVED** [review]

**Pull request merged**